File: typescript/7kyu/disemvowel/disemvowel.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Kata = void 0;
class Kata {
    static disemvowel(str) {
        return str.split("").filter(c => !"aeouiAEIOU".includes(c)).join("");
    }
}
exports.Kata = Kata;
// export class Kata {
//     static disemvowel(str: string) {
//         return str.replace(/[aeiou]/gi, "");
//     }
// }
const chai_1 = require("chai");
describe("disemvowel", function () {
    it("should pass a sample test", function () {
        chai_1.assert.strictEqual(Kata.disemvowel("This website is for losers LOL!"), "Ths wbst s fr lsrs LL!");
    });
});

File: python/7kyu/disemvowel.py
import codewars_test as test


def disemvowel(string_: str) -> str:
    return "".join("" if c.lower() in "aeiou" else c for c in string_)


@test.describe("Fixed Tests")
def basic_tests():
    @ test.it("First fixed test")
    def fixed_test_1():
        test.assert_equals(disemvowel(
            "This website is for losers LOL!"), "Ths wbst s fr lsrs LL!")
